src/span_context.js

```diff
@@ -134,7 +134,7 @@ export default class SpanContext {
     this._spanIdStr = null;
   }
 
-  set parentId(parentId: Buffer): void {
```

Comment:
Necessary because of https://github.com/jaegertracing/jaeger-client-node/blob/master/src/tracer.js#L226